### ISSUE TITLE
Space bar

### ISSUE THREAD

**Issue by sos0** [maxmarinich/react-alice-carousel#40]
Hi Max,

I currently have an unexpected behavior while using carousel in the same page as a semantic-ui-react search component where if I type a space bar in the search component, the carousel begins to auto play. Do you have any idea how I can remedy this? I don't see anywhere in the docs about managing space bar controls and I already have tried autoplay disabled (default and prop change didnt make a difference).

Thanks again.

**Comment by sos0** [comment]
For reference, here's my Search settings:
```
<Search
          category
          icon={isLoading ? LoadingIcon : SearchIcon}
          loading={isLoading}
          onResultSelect={this.handleResultSelect}
          onSearchChange={this.handleSearchChange}
          results={this.state.results}
          categoryRenderer={this.renderResultCategory}
          resultRenderer={this.renderResultItems}
          value={value}
        />
```

**Comment by maxmarinich** [comment]
Hi, @sos0! Please, provide full example.

**Comment by sos0** [comment]
Hi Max, I wasn't sure how to use codesandbox but I believe you can still see the repro here when you type something in the search then press space bar.
https://codesandbox.io/s/1zy47k0mvl

**Comment by sos0** [comment]
This is funny, but I tried blocking the space bar for the parent component with event.preventDefault and the space bar still triggered the autoplay for the carousel and blocked the search bar from entering spaces.

**Comment by maxmarinich** [comment]
@sos0, i can't reproduce the issue. Bu i've noticed some strange things: 
 - to run the example in the sandbox I had to update the dependencies(`react, react-dom` -> 16.0.0). 
 - `this.Carousel.state.items` - reading state form the refs
 - `const items = this.props.items.map(this.galleryItem);` - you're creating `items` in the `render`, every time the Gallery get new props and runs the recalculation
 - you always have the same properties `items.length === 1`
 - `responsive={{ 1024: { items: 3 } }}` - every time the Gallery get new props and runs the recalculation

Could you fix that problems and try reproduce the issue  in sandbox and update an example link?

**Comment by sos0** [comment]
@maxmarinich You can't reproduce the issue in the current sandbox? I updated the dependencies and typed "sldkjfs " (space at the end) and still was able to reproduce. Even if you don't type anything and click the browser window and press space bar, it will begin autoplay. (I'm on firefox by the way).

I don't have the line `const items = this.props.items.map(this.galleryItem)` that you're referring to in the sandbox so I'm a little confused. Please don't look at the code I wrote in the other issue because it's not relevant!

**Comment by sos0** [comment]
I see that the item length is one lol that was a copy/paste typo. Even if this is true, the error is reproducible!

**Comment by sos0** [comment]
Even without the semantic-ui's Search component, the basic-most example of the carousel will autoplay when you press spacebar in the window.

**Comment by maxmarinich** [comment]
fixed in [2b31006 ](https://github.com/maxmarinich/react-alice-carousel/commit/654bf342d1caff8eb36dae2ad6737e1060d567b6)
